setup.py

```diff
@@ -24,6 +24,7 @@
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
     ],
+    python_requires='>=3.8',
```

Comment:
@mauritsvanrees Adding this means that if someone tries to pip install Products.CMFPlone on an older Python, they'll get the most recent version that did not have this python_requires value.

Well, hopefully they are installing with a constraints file, and will see an error since the specified version isn't available for their Python. So it is probably okay. If they aren't using constraints, there are bigger problems.

Reply:
I have noticed that recent pip versions complain when you do not have python_requires at all.